server/sonar-server/src/main/java/org/sonar/server/user/UserSession.java

```diff
@@ -45,6 +45,13 @@
   @Deprecated
   Locale locale();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)